202404-big-data/2024-04-big-data.qmd

```diff
@@ -0,0 +1,856 @@
+---
+title: "Wrangling larger-than-memory data in R"
+author: "Renata Diaz"
+format: 
+  uaz-revealjs:
+      logo: "_extensions/cct-datascience/uaz/logo.png"
```

Comment:
The "new" Arizona Experiment station is in the root of the repo, so you can use:

```suggestion
      logo: "../logo.png"
```